src/include/processor/operator/copy_csv/copy_csv.h

```diff
@@ -30,19 +28,20 @@ class CopyCSV : public PhysicalOperator {
 protected:
     void errorIfTableIsNonEmpty(TablesStatistics* tablesStatistics) {
         auto numTuples = tablesStatistics->getReadOnlyVersion()
-                             ->tableStatisticPerTable.at(tableSchema.tableID)
+                             ->tableStatisticPerTable.at(tableID)
                              ->getNumTuples();
         if (numTuples > 0) {
+            auto tableName = catalog->getReadOnlyVersion()->getTableSchema(tableID)->tableName;
             throw CopyCSVException(
                 "COPY CSV commands can be executed only on completely empty tables. Table: " +
-                tableSchema.tableName + " has " + to_string(numTuples) + " many tuples.");
+                tableName + " has " + to_string(numTuples) + " many tuples.");
```

Comment:
Up to u but I guess we don't need to print tableName in return/error msg. After all, the msg points to a specific ddl/copyCSV statement which doesn't have any ambiguity. This comment applies to all DDL and CopyCSV operators.

If so, you can get rid of the code `auto tableName = catalog->getReadOnlyVersion()->getTableSchema(tableID)->tableName;`

Reply:
I think this may be helpful for debugging. It can quickly tell you which statement/table has failed.